### DISCUSSION TITLE
JavaScriptのコードブロックで文法上問題ないのに赤線が表示されることがある

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#351] [要望・アイデア・不具合]
たとえば以下のようなコードブロックを書くと、`\`部分に赤線が表示されます。

````md
```js
/^\d+$/.test('123')
```
````

<img width="796" alt="スクリーンショット 2022-11-09 21 30 10" src="https://user-images.githubusercontent.com/20086673/200830765-7bcbc23c-8fc9-4494-828d-826a8b247990.png">

GitHubではそうはなりません。

<img width="619" alt="スクリーンショット 2022-11-09 21 30 57" src="https://user-images.githubusercontent.com/20086673/200830925-80e7580f-a970-4a58-b6ff-44f2e38b86d1.png">

**Comment by wataru86** [comment]
@munierujp
ご意見ありがとうございます🙏

Qiita のハイライトは Rouge を使用しており、こちらの構文を Rouge がエラーと判定しているため赤線が出てしまいます...
http://rouge.jneen.net/v4.0.0/javascript/L15cZCskLy50ZXN0KCcxMjMnKQ

Qiita 側で赤線を消すことは可能ですが、他のエラーのハイライトにも影響が出るため難しいと考えています。
Rouge 側の修正も含め、この問題を修正できないか検討いたします 🙇
